kie-server-parent/kie-server-services/kie-server-services-jbpm-queries/src/test/java/org/kie/server/services/jbpm/queries/JbpmQueriesKieServerExtensionTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.kie.server.services.jbpm.queries;
+
+import static org.junit.Assert.*;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.ServiceLoader;
+
+import org.jbpm.services.api.query.QueryService;
+import org.junit.Test;
+import org.kie.server.services.api.KieServerApplicationComponentsService;
+import org.kie.server.services.api.KieServerRegistry;
+import org.kie.server.services.api.SupportedTransports;
+import org.mockito.Mockito;
+
+public class JbpmQueriesKieServerExtensionTest {
```

Comment:
Depends on who "we" are. I just started adding tests for the major components. This one simply tests the "appComponents" method to validate it returns what we expect it to return. Not sure if that is very valuable.